test/Makefile

```diff
@@ -15,12 +15,13 @@ GOFLAGS =
 all: deps
 
 global-deps:
-	make -C .. deps
+	$(MAKE) -C .. deps
 
 deps: global-deps bins
 
 clean:
-	rm $(BINS)
+	$(MAKE) -C sharness clean
+	-rm -rf $(BINS)
```

Comment:
Okay then I think it's not needed -- `-f` makes it ignore non-existing files/dirs anyhow